testutil/src/main/java/org/spine3/test/NullToleranceTest.java

```diff
@@ -356,10 +392,42 @@ public Builder excludeMethod(String methodName) {
         /**
          * Adds the default value for the method argument.
          *
-         * <p> For example, if will be added the text line,
-         * that line will be passed as the method argument for the {@code String} parameter type.
-         * During the check that default value will be passed as the method argument every time,
-         * when the particular argument is not checking.
+         * <p> Since the some classes have no value by default, that method serves to add it.
+         *
+         * <p> The list of the classes and their default values:
+         * <ul>
+         *     <li> the {@code 0} for the {@code Byte};
+         *     <li> the {@code 0} for the {@code Short};
+         *     <li> the {@code 0} for the {@code Integer};
+         *     <li> the {@code 0} for the {@code Long};
+         *     <li> the {@code '\u0000'} for the {@code Character};
+         *     <li> the {@code 0.0} for the {@code Float};
+         *     <li> the {@code 0.0} for the {@code Double};
+         *     <li> the {@code false} for the {@code Boolean};
+         *     <li> the {@code 0} for the {@code byte};
+         *     <li> the {@code 0} for the {@code short};
+         *     <li> the {@code 0} for the {@code int};
+         *     <li> the {@code 0} for the {@code long};
+         *     <li> the {@code '\u0000'} for the {@code char};
+         *     <li> the {@code 0.0} for the {@code float};
+         *     <li> the {@code 0.0} for the {@code double};
+         *     <li> the {@code false} for the {@code boolean};
+         *     <li> the instance provided by the {@code getDefaultInstance} method
+         *     will be the default instance for the {@code GeneratedMessageV3};
+         * </ul>
+         *
+         * <p> For example:
+         * {@code public static void method(GeneratedMessageV3 message, int primitive, CustomType obj)}.
```

Comment:
Instead of `GeneratedMessageV3 message` please introduce type, that looks realistic: either use a Message-based type declared in Spine already  or declare the type right here in the JavaDoc section.